packages/paste-token-designer/components/preview/components/AlertPreview.js

```diff
@@ -0,0 +1,29 @@
+import {Alert} from '@twilio-paste/core/alert';
```

Comment:
nit: the folder structure of `components/preview/components` is a little awkward.  Would it be fine to move all of these up a level?